pkg/kubectl/cmd/stop.go

```diff
@@ -46,7 +49,7 @@ func (f *Factory) NewCmdStop(out io.Writer) *cobra.Command {
 		Filenames util.StringList
 	}{}
 	cmd := &cobra.Command{
-		Use:     "stop (<resource> <id>|-f filename)",
+		Use:     "stop (<resource> (<id> | -l <label> | --all) | -f filename)",
```

Comment:
Please make the usage message consistent with delete:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/delete.go#L63